packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,389 @@
+              onPress={onPressNavigationButton}
+              text={t('dappsScreenBottomSheet.button', { dappName: dappSelected?.name })}
+            />
+          </View>
+        </View>
+      </BottomSheet>
+
+      <Dialog
+        title={t('dappsScreenHelpDialog.title')}
+        isVisible={isHelpDialogVisible}
+        secondaryActionText={t('dappsScreenHelpDialog.dismiss')}
+        secondaryActionPress={() => setHelpDialogVisible(false)}
+      >
+        {t('dappsScreenHelpDialog.message')}
+      </Dialog>
+
+      <ScrollView style={styles.scrollContainer}>
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading && error && (
+            <View style={{ ...styles.centerContainer, marginTop: 96 }}>
+              <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+            </View>
+          )}
+          {!loading && !error && result && (
+            <SectionList
+              style={{ flex: 1, padding: 16 }}
+              sections={parseResultIntoSections(result)}
+              renderItem={({ item: category }) => (
+                <Dapp dapp={category} onPressDapp={onPressDapp} />
+              )}
+              keyExtractor={(item: Dapp) => `${item.categoryId}-${item.id}`}
+              renderSectionHeader={({
+                section,
+              }: {
+                section: SectionListData<any, SectionData>
+              }) => <CategoryHeader category={section.category} />}
+            />
+          )}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function parseResultIntoSections(categoriesWithDapps: CategoryWithDapps[]): SectionData[] {
+  return categoriesWithDapps.map((category) => ({
+    data: category.dapps,
+    category: category,
+  }))
+}
+
+function CategoryHeader({ category }: { category: CategoryWithDapps }) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(category)}`)
+
+  return (
+    <View style={styles.categoryContainer}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: category.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: category.fontColor }}>{category.name}</Text>
+      </View>
+    </View>
+  )
+}
+
+function Dapp({ dapp, onPressDapp }: DappProps) {
+  Logger.debug(TAG, `Render item ${JSON.stringify(dapp)}`)
+
+  const onPress = () => {
```

Comment:
super nit: I think this would look slightly nicer as `const onPress = () => onPressDapp(dapp)`